### DISCUSSION TITLE
Missing year in agent.log

### DISCUSSION THREAD

**Discussion by thomas-merz** [puppetlabs/puppet-agent#2217] [Q&A]
Does anybody else of the Puppet users/admins is missing the year in agent.log? I knew a time when agen.log had year in it, so when log was older than a year and someone grep`ed or search for a specific date it was clear what year this log entry belongs to. Now it's only "Month Day" 😞

**Comment by joshcooper** [comment]
When puppet is configured to log to a file, I see the following time format:

```
$ bundle exec puppet agent --no-daemonize --onetime --logdest /tmp/puppet.log

$ head /tmp/puppet.log 
2024-04-23 15:00:10 -0700 Puppet (notice): Starting Puppet client version 8.7.0
2024-04-23 15:00:10 -0700 Puppet (err): Connection to https://localhost:8140/puppet-ca/v1 failed, trying next route: Request to https://localhost:8140/puppet-ca/v1 failed after 0.001 seconds: Failed to open TCP connection to localhost:8140 (Connection refused - connect(2) for "localhost" port 8140)
```

**Reply by thomas-merz** [reply]
That's true and strange at the same time:  

`puppet agent --no-daemonize --onetime --config /etc/puppetlabs/puppet/puppet.conf --logdest /tmp/puppet.log` produces nice ISO timestamps in the logdest-logfile:
> 2024-04-24 09:50:41 +0200 Puppet (notice): Applied catalog in 15.65 seconds


`puppet agent --no-daemonize --onetime --config /etc/puppetlabs/puppet/puppet.conf` doesn't produce ISO timestamps in my `agent.log` in `logdir = /var/log/puppetlabs/puppet` in `[main]` section from my config file 🤷🏼‍♂️
> Apr 24 09:53:11 tst-thomas-08 puppet-agent[3072263]: Applied catalog in 16.17 seconds

Any idea why this is acting so weird? (agents are currently on 7.30.0)
